lib/esc.js

```diff
@@ -394,17 +379,7 @@ ESC.prototype.brake = function() {
  * @return {Object} instance
  */
 ESC.prototype.stop = function() {
-  var state = priv.get(this);
-  var history = state.history;
-  var speed = this.type === "bidirectional" ? this.neutral : 0;
-
-  this.write(this.pin, Fn.fscale(speed, 0, 100, this.pwmRange[0], this.pwmRange[1]));
-
-  history.push({
-    timestamp: Date.now(),
-    speed: speed
-  });
-
+  this.write(this.pin, this.neutral);
```

Comment:
`stop` and `brake` were basically the same thing... Let's figure out why

Reply:
I have a guess why it's like this. Some ESC's have the ability to brake and some don't (that's speculation, not fact). The default don't so they just replicate the behavior of ```stop```. That can be overridden in controller objects that do support it.